src/LibChorus/FileTypeHandlers/ChorusFileTypeHandlerCollection.cs

```diff
@@ -44,8 +44,21 @@ public class ChorusFileTypeHandlerCollection
 
 				using (var container = new CompositionContainer(catalog))
 				{
-					container.ComposeParts(this);
-				}
+                    try
+                    {
+                        container.ComposeParts(this);
+                    }
+                    catch (Exception ex)
+                    {
+                        if (ex is ReflectionTypeLoadException)
```

Comment:
Wouldn't it be better to directly catch the `ReflectionTypeLoadException` instead of all exceptions?